lib/metric_collector/native/radon/parser/cyclomatic.rb

```
@@ -0,0 +1,62 @@
+module MetricCollector
+  module Native
+    module Radon
+      module Parser
+        class Cyclomatic < MetricCollector::Native::Radon::Parser::Base
+          def self.command
+            'cc'
+          end
+
+          def self.default_value
+            1.0
+          end
+
+          def self.parse(collected_metrics_hash, processing, metric_configuration)
+            self.results_enumerator(collected_metrics_hash).each do |module_name, result, granularity|
+              module_name = module_name + '.' + result['name']
+              module_result = module_result(module_name, granularity, processing)
+              value = result['complexity'].to_f
+              MetricResult.create(metric: metric_configuration.metric, value: value, module_result: module_result,
+                                  metric_configuration_id: metric_configuration.id)
+            end
+          end
+
+          private
+
+          def self.results_enumerator(collected_metrics_hash)
```

This method seems rather complicated. Isn't there a better way to do the parsing?
If there is not, some other things must be reviewed:
 * `y` is not a very good variable name
 * Is it not better to raise an exception instead of writing a debug message if there is an error on Radon's execution?